### PR TITLE
Test case for PBM-1096

### PR DESCRIPTION
**Pull request by sandraromanchenko**

### CODE REVIEW

File: pbm-functional/pytest/test_azurite.py [hunk 1]
```diff
@@ -68,6 +68,34 @@ def test_logical(start_cluster, cluster):
     assert pymongo.MongoClient(cluster.connection)["test"].command("collstats", "test").get("sharded", False)
     Cluster.log("Finished successfully\n")
 
+@pytest.mark.timeout(300, func_only=True)
```
Comment (olexandr-havryliak) [inline pbm-functional/pytest/test_azurite.py:71]: Increase timeout for the test

File: pbm-functional/pytest/test_sharded.py [hunk 2]
```diff
@@ -69,6 +69,34 @@ def test_logical(start_cluster,cluster):
     assert pymongo.MongoClient(cluster.connection)["test"].command("collstats", "test").get("sharded", False)
     Cluster.log("Finished successfully")
 
+@pytest.mark.timeout(300, func_only=True)
```
Comment (olexandr-havryliak) [inline pbm-functional/pytest/test_sharded.py:72]: Increase timeout for the test

Comment (sandraromanchenko) [reply pbm-functional/pytest/test_sharded.py]: Done)

File: pbm-functional/pytest/test_azurite.py (hunk 1 above)
Comment (sandraromanchenko) [reply pbm-functional/pytest/test_azurite.py]: Done)

**Review by olexandr-havryliak — APPROVED** [review]

**Pull request merged**